include/my_sys.h

```diff
@@ -487,46 +488,101 @@ typedef void (*my_error_reporter)(enum loglevel level, const char *format, ...)
 
 extern my_error_reporter my_charset_error_reporter;
 
-/* defines for mf_iocache */
+/* inline functions for mf_iocache */
 extern PSI_file_key key_file_io_cache;
 
+MY_NODISCARD
+extern int _my_b_get(IO_CACHE *info);
+MY_NODISCARD
+extern int _my_b_read(IO_CACHE *info, uchar *Buffer, size_t Count);
+MY_NODISCARD
+extern int _my_b_write(IO_CACHE *info, const uchar *Buffer, size_t Count);
+
 /* Test if buffer is inited */
```

Comment:
The comment is misplaced, should go to my_b_inited (might as well report to MariaDB, in batch with any other comments)

Reply:
fixed